mora/app.py

```diff
@@ -24,8 +26,19 @@
 app = flask.Flask(__name__, root_path=distdir, template_folder=templatedir)
 app.cli = cli.group
 
+# Session setup
+app.config.update({
+    'SESSION_TYPE': 'filesystem',
+    'SESSION_PERMANENT': False,
+    'SESSION_FILE_DIR': settings.SESSION_FILE_DIR
+})
+Session(app)
```

Comment:
This is an example of why I prefer module-level imports rather than individual fields. `Session` is a rather general name, also used by `requests`, for example, whereas `flask_session.Session()` is quite unambiguous.